examples/tutorial_client_eventfilter.c

```diff
@@ -0,0 +1,267 @@
+        if (!contentFilter[0].elements[i].filterOperands){
+            UA_ContentFilter_clear(contentFilter);
+            return NULL;
+        }
+        for(size_t n =0; n< contentFilter[0].elements[i].filterOperandsSize; ++n) {
+            UA_ExtensionObject_init(&contentFilter[0].elements[i].filterOperands[n]);
+        }
+    }
+    /* first Element (OR) */
+    contentFilter->elements[0].filterOperands[0].content.decoded.type = &UA_TYPES[UA_TYPES_ELEMENTOPERAND];
+    contentFilter->elements[0].filterOperands[0].encoding = UA_EXTENSIONOBJECT_DECODED;
+    contentFilter->elements[0].filterOperands[1].content.decoded.type = &UA_TYPES[UA_TYPES_ELEMENTOPERAND];
+    contentFilter->elements[0].filterOperands[1].encoding = UA_EXTENSIONOBJECT_DECODED;
+    UA_ElementOperand *firstElementOperand = UA_ElementOperand_new();
+    UA_ElementOperand_init(firstElementOperand);
+    firstElementOperand->index = 1;
+    UA_ElementOperand *secondElementOperand = UA_ElementOperand_new();
+    UA_ElementOperand_init(secondElementOperand);
+    secondElementOperand->index = 2;
+    contentFilter->elements[0].filterOperands[0].content.decoded.data = firstElementOperand;
+    contentFilter->elements[0].filterOperands[1].content.decoded.data = secondElementOperand;
+    /* second Element (OfType) */
+    contentFilter[0].elements[1].filterOperands[0].content.decoded.type = &UA_TYPES[UA_TYPES_LITERALOPERAND];
+    contentFilter[0].elements[1].filterOperands[0].encoding = UA_EXTENSIONOBJECT_DECODED;
+    UA_LiteralOperand *literalOperand = UA_LiteralOperand_new();
+    UA_LiteralOperand_init(literalOperand);
+    UA_NodeId *nodeId = UA_NodeId_new();
+    UA_NodeId_init(nodeId);
+    nodeId->namespaceIndex = 0;
+    nodeId->identifierType = UA_NODEIDTYPE_NUMERIC;
+    nodeId->identifier.numeric = UA_NS0ID_AUDITEVENTTYPE;
+    UA_Variant_setScalar(&literalOperand->value, nodeId, &UA_TYPES[UA_TYPES_NODEID]);
+    contentFilter[0].elements[1].filterOperands[0].content.decoded.data = literalOperand;
+    /* third Element (OfType) */
+    contentFilter[0].elements[2].filterOperands[0].content.decoded.type = &UA_TYPES[UA_TYPES_LITERALOPERAND];
+    contentFilter[0].elements[2].filterOperands[0].encoding = UA_EXTENSIONOBJECT_DECODED;
+    UA_LiteralOperand *secondLiteralOperand = UA_LiteralOperand_new();
+    UA_LiteralOperand_init(secondLiteralOperand);
+    UA_NodeId *secondNodeId = UA_NodeId_new();
+    UA_NodeId_init(secondNodeId);
+    secondNodeId->namespaceIndex = 0;
+    secondNodeId->identifierType = UA_NODEIDTYPE_NUMERIC;
+    secondNodeId->identifier.numeric = UA_NS0ID_EVENTQUEUEOVERFLOWEVENTTYPE;
+    UA_Variant_setScalar(&secondLiteralOperand->value, secondNodeId, &UA_TYPES[UA_TYPES_NODEID]);
+    contentFilter[0].elements[2].filterOperands[0].content.decoded.data = secondLiteralOperand;
+    return contentFilter;
+}
+
+static UA_SimpleAttributeOperand *
+setupSelectClauses(void) {
```

Comment:
The "setupWhereClauses" has a description which is missing here.

Reply:
We added the description in the next commit.